internal/containerizedengine/engine.go

```diff
@@ -89,35 +88,6 @@ func (c baseClient) getEngineImage(engine containerd.Container) (string, error)
 	return image.Name(), nil
 }
 
-// getEngineConfigFilePath will extract the config file location from the engine flags
-func (c baseClient) getEngineConfigFilePath(ctx context.Context, engine containerd.Container) (string, error) {
```

Comment:
Originally this was in use, but in the refactorings it wasn't necessary.  My suspicion is we may need this again as things are fleshed out further in the future, but we can pull it back into the main package at that time.

Reply:
yes exactly :wink: